src/network/network_lan_promotion.cc

```diff
@@ -178,7 +178,7 @@ bool LanBase::is_open() {
 	return socket_v4.is_open() || socket_v6.is_open();
 }
 
-ssize_t LanBase::receive(void* const buf, size_t const len, NetAddress* addr) {
+size_t LanBase::receive(void* const buf, size_t const len, NetAddress* addr) {
```

Comment:
Look for `if (receive(&info, sizeof(info), &addr) < static_cast<int32_t>(sizeof(info))) {` in function `void LanGameFinder::run()` and remove the `static_cast`, as this return type is no longer signed.